.github/workflows/scorecard.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
       # actions: read
 
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@v2
+        with:
+          egress-policy: audit
       - name: "Checkout code"
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
We use checkout@v4 elsewhere, we should do the same here.

Reply:
I can do that, just have to silence it too 👍🏼